shell/src/main/java/alluxio/shell/command/StatCommand.java

```diff
@@ -25,45 +24,48 @@
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * Displays the file's all blocks info.
+ * Displays the path's info.
+ * If path is a directory it displays the directory's info.
+ * If path is a file, it displays the file's all blocks info.
  */
 @ThreadSafe
-public final class FileInfoCommand extends WithWildCardPathCommand {
+public final class StatCommand extends WithWildCardPathCommand {
```

Comment:
We should not simply rename `fileInfo` to `stat`. Instead, we should keep the old `fileInfo` command around, and have it print information similar to what the `loadMetadata` command does informing the user that the command is deprecated and what command should be used instead.